### PR TITLE
Add PyPa virtualenv/build

### PR DESCRIPTION
**Pull request by gaborbernat**
👍🏻

### CODE REVIEW

File: communities.json [hunk 1]
```diff
@@ -771,6 +771,13 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "pypa.png",
```
Comment (jamiebuilds) [inline communities.json:776]: Can you run this image through a PNG optimizer?

File: communities.json [hunk 2]
```diff
@@ -771,6 +771,13 @@
...
+      "title": "PyPa",
+      "quote": "Some Python Packaging Authority projects (such as virtualenv) use Discord to answer questions from users and to collaborate on development tasks.",
+      "inviteCode": "gDvaHVJvsn",
```
Comment (jamiebuilds) [inline communities.json:779]: Is this an official server that you're just settings up. There is only 1 member in this community

Screen Shot 2021-02-10 at 10 13 33 AM

Comment (gaborbernat) [reply communities.json:779]: Yeah, it's a new server. I'm the maintainer of the virtualenv package, for which I would like to move to discord. (and other projects might join in, some still prefer to use irc).

File: communities.json (hunk 1 above)
Comment (gaborbernat) [reply communities.json:776]: Done.

**Comment by jamiebuilds** [comment]
Turned on some features for you:

- [Vanity Invite URL](https://support.discord.com/hc/en-us/articles/115001542132-Server-Vanity-URLs)
- [Server Banner & Invite Splash](https://support.discord.com/hc/en-us/articles/360028716472-Server-Banner-Background-Invite-Splash-Image)

If you'd like, you can go update your invite code to something like `discord.gg/<yourprojectname>` and then update it in this PR. Either way, please let me know.

**Comment by gaborbernat** [comment]
Updated, thanks. All good to go.

**Review by jamiebuilds — APPROVED** [review]

**Comment by gaborbernat** [comment]
Guess we can close this then?

**Pull request closed without merging**